servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/H1SpecExceptions.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright © 2020 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.http.netty;
+
+/**
+ * Additional extensions for <a href="https://tools.ietf.org/html/rfc7230">HTTP/1.1</a> specification that help to
+ * relax constrains for backward compatibility with older systems.
+ */
+public interface H1SpecExceptions {
```

Comment:
Since this is just a collection of flags and no behavior, I would suggest to make this a concrete class with a private constructor and have the builder as an inner class. It helps to reduce the level of indirections to create an instance.